bigquery/google/cloud/bigquery/client.py

```diff
@@ -525,6 +544,171 @@ def load_table_from_storage(self, source_uris, destination,
         job.begin()
         return job
 
+    def load_table_from_file(self, file_obj, destination,
+                             rewind=False,
+                             size=None,
+                             num_retries=_DEFAULT_NUM_RETRIES,
+                             job_id=None, job_config=None):
+        """Upload the contents of this table from a file-like object.
+
+        Like load_table_from_storage, this creates, starts and returns
+        a ``LoadJob``.
+
+        :type file_obj: file
+        :param file_obj: A file handle opened in binary mode for reading.
+
+        :type destination: :class:`google.cloud.bigquery.table.TableReference`
+        :param destination: Table into which data is to be loaded.
+
+        :type rewind: bool
+        :param rewind: If True, seek to the beginning of the file handle before
+                       writing the file.
+
+        :type size: int
+        :param size: The number of bytes to read from the file handle.
+                     If not provided, we'll try to guess the size using
```

Comment:
I'm not sure that the `os.fstat` comment is correct, since I'm not seeing it here. I think this argument should talk about resumable vs multi-part upload since that's what it's being used for as far as I can tell.

Reply:
Done.